package.json

```diff
@@ -46,6 +46,7 @@
     "react-native": "^0.67.4",
     "react-native-webview": "^11.17.2",
     "typescript": "^4.5.4",
-    "vite": "^2.8.0"
+    "vite": "^2.8.0",
```

Comment:
Could we maybe upadate this to `5.0`? That's the version we're using elsewhere. I would update that package in particular (and leave the others the version they already are)

Reply:
Updated and building correctly! 🙌🏼